test/unit/membership-test.js

```diff
@@ -31,7 +31,38 @@ describe('Membership', () => {
       expect(result.message[0]).to.eql('See the notFound list for failed member puts');
     });
 
-    it('should add multiple members to a group', async () => {
+    it('should replace entire membership of a group', async () => {
+      const data =
+      {
+        data: [
+          {
+            id:   'clooper',
+            type: 'uwnetid',
+          },
+          {
+            id:   'hc75',
+            type: 'uwnetid',
+          },
+          {
+            id:   'gabugabu',
+            type: 'uwnetid',
+          }
+        ]
```

Comment:
maybe add a length test so that you make sure you get something other than 0 results?

Reply:
I added a test that checks for a 404 when the data array doesn't contain any members to add

Reply:
I added a test that the length of the data array in options matches the length of the returned membership data - to ensure that all members are correctly replaced.